Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel.yml

```diff
@@ -74,6 +74,12 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Timeout
+  name: timeout
+  additionalinfo: in seconds
+  hidden: false
+  required: false
+  type: 0
```

Comment:
You didn't implement this parameter. Should it be removed?

Reply:
yup